docs/DeveloperGuide.md

```diff
@@ -3,22 +3,30 @@ layout: page
 title: Developer Guide
 ---
 * Table of Contents
-{:toc}
+- [Acknowledgements](#acknowledgements)
+- [Setting Up](#setting-up)
+- [Design](#design)
+- [Implementation](#implementation)
+- [Misc. Docs](#misc-documentation)
+- [Appendix](#appendix)
 
 --------------------------------------------------------------------------------------------------------------------
 
+<a id="acknowledgements"></a>
```

Comment:
The Generated TOC links to eac header also, so this doesn't need to be done too.